_P176_CDM7160.ino

```diff
@@ -236,74 +171,72 @@ boolean Plugin_176(byte function, struct EventStruct *event, String& string)
       }
   case PLUGIN_INIT:
       {
-      plugin_176_setPowerDown(); // disabled because there were issues setting alt in powerdown mode
-      unsigned char elev = Settings.TaskDevicePluginConfig[event->TaskIndex][1] / 10;
-      delay(100);
-      if (elev)
-      {
-       plugin_176_setAltitude(elev);
-      } else
-      {
-       plugin_176_clearAltitude();
-      }
-      delay(100);
-      plugin_176_setContinuous();
-      success = true;
+        plugin_176_setPowerDown(); // disabled because there were issues setting alt in powerdown mode
+        unsigned char elev = Settings.TaskDevicePluginConfig[event->TaskIndex][1] / 10;
+        delay(100);
```

Comment:
If it will block other I2C communications, then it's another matter. Also since it is the init function, it will not be happening that often.
Please add this reason as a comment, or else someone (probably me) will try to change it and see why it should be using the delay.

Reply:
Comment was added.